cmd/microcloud/ask.go

```diff
@@ -1138,7 +1138,14 @@ func (c *initConfig) askOVNNetwork(sh *service.Handler) error {
 		}
 	}
 
-	var ovnUnderlaySelectedIPs map[string]string
+	// ipWithIface is a helper struct to store the selected IP address and
+	// the corresponding network interface in one element.
+	type ipWithIface struct {
+		ip        string
```

Comment:
Should the IP rather be stored in raw format (I guess `net.IP`) instead of just string? Might become useful later if we use the struct for additional use cases.

Reply:
Yes good idea! fixed